### PR TITLE
14247 - NPE in FastMatch when CurrentZone returns null

### PR DESCRIPTION
**Pull request by Cattlesquat**
…ome back null for a piece

### CODE REVIEW

File: vassal-app/src/main/java/VASSAL/counters/GlobalCommand.java [hunk 1]
```diff
@@ -467,25 +467,25 @@ else if (!target.targetType.isCurrent() && !fastMap.isEmpty() && !fastMap.equals
                   switch (target.targetType) {
                   case ZONE:
                   case CURZONE:
-                    if (!fastZone.equals(gamePiece.getProperty(BasicPiece.CURRENT_ZONE))) {
+                    if ((fastZone != null) && !fastZone.equals(gamePiece.getProperty(BasicPiece.CURRENT_ZONE))) {
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/counters/GlobalCommand.java:470]: Should this to be `fastZone == null ||` instead? If I understand this right, the `continue` case is for a non-match.

Comment (Cattlesquat) [reply vassal-app/src/main/java/VASSAL/counters/GlobalCommand.java:470]: done

**Comment by uckelman** [comment]
Waiting for user to reply to @Cattlesquat.

**Comment by BrentEaston** [comment]
Brian, Is this ready to go? It seems like a fairly straight-forward defensive programming change.

**Comment by BrentEaston** [comment]
Also, it likes the cause is that fastZone is returning null, not CurrentZone. Is there a problem in our Configurer?

**Comment by Cattlesquat** [comment]
Oh actually this can be close now (and 14247) because of https://github.com/vassalengine/vassal/pull/872

**Pull request closed without merging**